server/views/pages/exhibition.njk

```diff
@@ -39,7 +39,9 @@
     <div class="container">
       <div class="grid">
         <div class="{{ {s:12, m:10, shiftM:1, l:7, xl:7} | gridClasses }} {{ {s:'HNL3', m:'HNL2'} | fontClasses}} {{ {s:4} | spacingClasses({margin: ['bottom']}) }}">
-          {% block exhibitionDescription %}{% endblock %}
+          {% block exhibitionDescription %}
```

Comment:
But then I would need to update Cardigan with mock data, which I am a little loathed to do.